zerver/lib/bugdown/help_settings_links.py

```diff
@@ -0,0 +1,85 @@
+import re
+import markdown
+from typing import Any, Dict, List, Optional, Union, Text
+from typing.re import Match
+from markdown.preprocessors import Preprocessor
+
+REGEXP = re.compile(r'\{settings_tab\|(?P<setting_identifier>.*?)\}')
+
+link_mapping = {
+    # a mapping from the setting identifier that is the same as the final URL
+    # breadcrumb to that setting to the name of its setting type, the setting
+    # name as it appears in the user interface, and a relative link that can
+    # be used to get to that setting
+    'your-account': ['Settings', 'Your account', '/#settings/your-account'],
+    'display-settings': ['Settings', 'Display settings', '/#settings/display-settings'],
+    'notifications': ['Settings', 'Notifications', '/#settings/notifications'],
+    'emoji-settings': ['Manage organization', 'Custom emoji', '/#organization/emoji-settings'],
+}
+
+settings_markdown = """
+1. From your desktop, click on the **gear**
+   (<i class="icon-vector-cog"></i>) in the upper right corner.
+
+1. Select **%s**.
+
+1. On the left, click %s.
```

Comment:
I'd recommend replacing the `%s` with `%(variable_name)s` and passing a dict into `%` for string formatting rather than a tuple; that would create a much more readable result.